tests/network/hotplug.go

```diff
@@ -142,3 +163,24 @@ func interfaceStatusFromInterfaceNames(ifaceNames ...string) []v1.VirtualMachine
 	}
 	return ifaceStatus
 }
+
+func filterVMISyncErrorEvents(events []corev1.Event) []string {
+	const desiredEvent = "SyncFailed"
+	return filterEvents(events, func(event corev1.Event) bool {
+		return event.Reason == desiredEvent
+	})
+}
+
+func filterEvents(events []corev1.Event, p func(event corev1.Event) bool) []string {
+	var eventMsgs []string
+	for _, event := range events {
+		if p(event) {
+			eventMsgs = append(eventMsgs, event.Message)
+		}
+	}
+	return eventMsgs
+}
+
+func noPCISlotsAvailableError() string {
+	return "server error. command SyncVMI failed: \"LibvirtError(Code=1, Domain=20, Message='internal error: No more available PCI slots')\""
```

Comment:
This is fragile. Any change to the error message (in kubevirt or libvirt code) will fail this test and will cause us a lot of work.

Why do we need to check logs/events and not just something formal, like a condition or plan showing that the vNIC is not plugged (i.e. not in the VMI status).